sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -95,49 +95,85 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     catalog.createPartitions(tableName, Seq(part), ignoreIfExists = false)
   }
 
+  private def dropTable(catalog: SessionCatalog, name: TableIdentifier): Unit = {
+    catalog.dropTable(name, ignoreIfNotExists = false)
+  }
```

Comment:
It is being used by one test case. Yeah, I can move it into the test case, instead of creating a new function here.